app/assets/stylesheets/common.css.scss

```diff
@@ -73,3 +73,7 @@ body {
 .invoice-title {
   color: $title-invoice-color
 }
+
+#new_invitation {
```

Comment:
I see that form for create new Invite will have this margin too, I'm not sure if you need this.